enterprise/frontend/src/embedding-sdk/components/public/MetabotQuestion/MetabotQuestion.tsx

```diff
@@ -19,7 +22,16 @@ const MetabotQuestionInner = () => {
           title={false}
           onNavigateBack={() => {}}
           isSaveEnabled={false}
-        />
+        >
+          <InteractiveQuestionDefaultView
```

Comment:
This view matches the new design here https://www.figma.com/design/l9y9OEENQUYiKSbFo7WQA0/Embedding-Metabot-w--SDK-to-ask-questions-using-natural-language?node-id=17192-3680&t=IOa8LxyUdme586R7-0